wrappers/python/tests/TestForceField.py

```diff
@@ -300,6 +300,26 @@ def test_Forces(self):
                 numDifferences += 1
         self.assertTrue(numDifferences < system.getNumParticles()/20) # Tolerate occasional differences from numerical error
 
+    def test_ImplicitSolventForces(self):
+        """Compute forces for different implicit solvent types, and compare them to ones generated with AmberPrmtopFile."""
+
+        solventType = ['obc1', 'obc2', 'gbn2']
```

Comment:
This only tests 3 out of 5 models, does it not? Why not include complete test coverage?

Reply:
For HCT and GBn, the forces don't quite match because the new version uses different radii than the old one (mbondi for HCT, bondi for GBn).  That's to match the recommendations in the Amber manual.  I figured testing three of the files here was good enough, since the only difference between them is the class name and all the Python code is already tested elsewhere.

Reply:
Can we at lest test _applying_ the parameters to make sure a typo doesn't creep into the `xml` files? There is no way to ensure they remain valid otherwise.